src/filters/time/time.constants.ts

```diff
@@ -2,9 +2,43 @@ const PM = `pm|p.m.`;
 const AM = `am|a.m.`;
 const MERIDIEM = `( )?(${AM}|${PM})`;
 const FILLER_WORDS = '(at|by) ';
+const HOUR_PART = '[0-9]{1,2}';
+const MINUTE_PART = `(:[0-5][0-9])`;
+
+const HALF = 'half';
+const QUARTER = 'quarter';
+
+const VERBAL_QUANTIFIERS = {
+    ANY: `(${HALF}|${QUARTER})`,
+    HALF: `${HALF}`,
+    QUARTER: `${QUARTER}`,
+};
+
+const TO = `(to)`;
+const PAST = `(after|past)`;
+const MINUTE_IDENTIFIER = '( min| mins| minutes)';
```

Comment:
```suggestion
const MINUTE_IDENTIFIER = '( (min|mins|minutes))';
```

Reply:
Maybe we can remove the space from here, and add it in when we need to use this constant?

Reply:
I think it is easier to keep it here, adding this constant invariably requires space.